### DISCUSSION TITLE
Reproducible frontend crash with Anthropic Vertex AI Selection

### DISCUSSION THREAD

**Discussion by ArcCal** [danny-avila/LibreChat#4623] [Troubleshooting]
### What happened?

Trying to use claude via google vertext, librechat crashed.

### Steps to Reproduce

1.following the instructions on these page to try configuring Google Vertex AI 

(https://www.librechat.ai/docs/remote/docker_linux

and

https://www.librechat.ai/docs/configuration/pre_configured_ai/google). 

2. using caddy to revere proxy 

:8444 ai.examppppppple.com {

        tls  {
                dns cloudflare XXXXXXX
                client_auth {
                mode require_and_verify
                trusted_ca_cert_file /etc/ssl/mtls/ca-cert.pem
                }
        }

    reverse_proxy http://localhost:3080

}

3. I was able to use gemini-1.5-flash-002 and gemini-1.5-pro-002 successfully, but when I tried to use claude-3-5-sonnet-v2@20241022, it immediately caused LibreChat to crash, with the entire browser displaying the following error

### Relevant log output

```shell
Unexpected Application Error!
Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Blc%2C%20type%2C%20id%2C%20kwargs%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

gw@https://ai.examppppppple.com/assets/vendor.BDtCgmSV.js:38:12815
C@https://ai.examppppppple.com/assets/vendor.BDtCgmSV.js:38:17876
so@https://ai.examppppppple.com/assets/vendor.BDtCgmSV.js:40:1481
Qee@https://ai.examppppppple.com/assets/vendor.BDtCgmSV.js:40:44543
zee@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:40:39879
pSe@https://ai.examppppppple.com/assets/vendor.BDtCgmSV.js:40:39802
v2@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:40:39648
j$@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:40:36004
iB@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:40:36809
Kf@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:38:3288
Vo/<@https://ai.examppppple.com/assets/vendor.BDtCgmSV.js:40:34324
```

### Screenshots

<img width="1260" alt="image" src="https://github.com/user-attachments/assets/d540536a-00bb-4f63-8a66-b628bcb3a39d">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ArcCal** [comment]
It’s confirmed that only LibreChat crashes; Caddy and Docker remain unaffected. After refreshing the browser, LibreChat appears normal again. However, as soon as the claude-3-5-sonnet-v2@20241022 model is selected and any input is entered, it immediately crashes again.

**Comment by danny-avila** [comment]
Anthropic models are not supported via vertex. This is a frontend crash and I will attempt to reproduce it, but it wouldn’t work even if it didn’t crash

**Comment by ArcCal** [comment]
I applied to use Claude through the Model Garden in the Google AI Studio page, and I've been using it via API for several months now. So, I'm not quite sure what you mean by "Anthropic models are not supported via Vertex."

**Comment by ArcCal** [comment]
https://cloud.google.com/vertex-ai/generative-ai/docs/partner-models/use-claude

**Comment by Ziyann** [comment]
I submitted a PR draft for this, comments are welcome: https://github.com/danny-avila/LibreChat/pull/9798.
